addons/web/static/src/search/custom_group_by_item/custom_group_by_item.xml

```diff
@@ -1,24 +1,14 @@
 <templates xml:space="preserve">
 
     <t t-name="web.CustomGroupByItem">
-        <AccordionItem class="'o_add_custom_group_menu text-truncate'" description="'Add Custom Group'">
-            <div class="px-3 py-2">
-                <select class="pe-3 text-truncate o_input" t-model="state.fieldName">
-                    <option t-foreach="props.fields" t-as="field" t-key="field.name"
-                    t-att-value="field.name"
-                    t-if="field.type !== 'properties' and !field.isProperty"
-                    t-esc="field.string"
-                    />
-                </select>
-            </div>
-            <div class="px-3 py-2">
-                <button class="btn btn-primary w-100"
-                    t-on-click="() => props.onAddCustomGroup(state.fieldName)"
-                    >
-                    Apply
-                </button>
-            </div>
-        </AccordionItem>
+        <select class="o_add_custom_group_menu dropdown-item" t-on-change="(ev) => this.onSelected(ev)">
```

Comment:
why don't you add o_menu_item here also?

Reply:
don't forget to check that testFilters still works (clickbot.js)

Reply:
Okay, I'll add it

Reply:
In fact, I prefer not. In tests, it is annoying since the selectors would get the entire text of each options when reading the textContent (since it's a <select> with options inside